glide.lock

```diff
@@ -2030,7 +2030,7 @@ imports:
   subpackages:
   - config/v1beta1
 - name: k8s.io/kubernetes
-  version: de991a22fa8b3fb4d68abc87a064377cc793135a
+  version: c398de65fed4a7fbdc0b8a0c0682623d9dad41e1
```

Comment:
@deads2k were the kube bumps not supposed to happen? Seeing unit tests fail with panic.